### PR TITLE
Feat 07 html report

### PR DESCRIPTION
**Pull request by mbartelsm**
Improvements on the HTML report model

Co-Authored-By: JFaber151 <j.s.j.faber@student.rug.nl>

### CODE REVIEW

File: Faultify.Report/MutationStatus.cs [hunk 1]
```diff
@@ -3,8 +3,8 @@
     public enum MutationStatus
     {
         Survived,
-        Killed,
         Timeout,
         NoCoverage,
+        Killed,
```
Comment (cdadashovkhandan) [inline Faultify.Report/MutationStatus.cs:8]: Is there a reason for relocating this?

File: Faultify.Analyze/Analyzers/OpCodeAnalyzer.cs [hunk 2]
```diff
@@ -26,57 +27,44 @@ protected OpCodeAnalyzer(Dictionary<OpCode, IEnumerable<(MutationLevel, OpCode)>
...
         public IMutationGroup<OpCodeMutation> GenerateMutations(
-            Instruction scope,
+            MethodDefinition scope,
             MutationLevel mutationLevel,
             IDictionary<Instruction, SequencePoint> debug = null
         )
         {
-            OpCode original = scope.OpCode;
-            IEnumerable<OpCodeMutation> mutations;
+            var mutGroup = new List<IEnumerable<OpCodeMutation>>();
```
Comment (cdadashovkhandan) [inline Faultify.Analyze/Analyzers/OpCodeAnalyzer.cs]: Tiny nitpick but why not call it `MutationGroup` to maintain consistency and readability?

**Review by cdadashovkhandan — APPROVED** [review]
Great job! Got small nitpicks, but they're not important.

File: Faultify.Report/MutationStatus.cs (hunk 1 above)
Comment (mbartelsm) [reply Faultify.Report/MutationStatus.cs:8]: consistency, it has no practical effects, so I don't see the problem with the change.

Comment (cdadashovkhandan) [reply Faultify.Report/MutationStatus.cs:8]: Sure thing

**Pull request merged**